assignment.js

```diff
@@ -66,6 +71,12 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const values = [3, 48, 29, 12, 18]
+const values_sorted = values.sort(function(a, b){return b-a})
+const largest_value = values_sorted[0]
+
+console.log(largest_value)
+
 // Add your code above this line
 
```

Comment:
@sad3xoxo Nice Job on this problem using the array sort method to sort the values in descending order and print the largest value.
Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L73)